internal/worker/chain.go

```diff
@@ -98,7 +101,7 @@ func NewChain(
 }
 
 func (c *Chain) Start(ctx context.Context) error {
-	c.latestBlockNumber = c.cf.Chain.StartBlock
+	c.latestBlockNumber = c.cf.Chain.GetStartBlock()
```

Comment:
Service protocol already add verifier mapping: verifier_address -> block . Now we need to get verifier block from contract ?

Reply:
Will add this as a follow-up